facebook-commerce.php

```diff
@@ -1614,38 +1614,40 @@ public function delete_product_set_item( string $fb_product_set_id ) {
 
 	/**
 	 * Displays Batch API completed message on simple_product_publish.
-	 * This is called by the hook `add_meta_boxes` because that is sufficient time
+	 * This is called by the hook `add_meta_boxes_product` because that is sufficient time
 	 * to retrieve product_item_id for the product item created via batch API.
 	 *
 	 * Some sanity checks are added before displaying the message after publish
 	 *  - product_item_id : if exists, means product was created else not and don't display
 	 *  - should_sync: Don't display if the product is not supposed to be synced.
 	 *
-	 * @param string $post_type Wordpress Post type
 	 * @param WP_Post $post Wordpress Post
 	 * @return void
 	 */
-	public function display_batch_api_completed( string $post_type, WP_Post $post ) {
-		$fb_product     = new \WC_Facebook_Product( $post->ID );
-		$fb_product_item_id  = null;
-		$should_sync    = true;
-		$no_sync_reason = '';
+	public function display_batch_api_completed( $post ) {
```

Comment:
`add_meta_boxes_{$post_type}` sends only the `$post` object:
https://github.com/WordPress/wordpress-develop/blob/87dfd5514b52aef456b7232b1959873e69e651da/src/wp-admin/includes/meta-boxes.php#L1731-L1731